app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/PlanWizardInstancePropertiesStepTable.js

```diff
@@ -11,6 +10,22 @@ import { paginate, Grid, PaginationRow, Table, PAGINATION_VIEW, Icon, Button, Fo
 import TableInlineEditRow from './TableInlineEditRow/TableInlineEditRow';
 
 class PlanWizardInstancePropertiesStepTable extends React.Component {
+  componentWillUnmount() {
+    const { input, rows } = this.props;
+    const minimalInstancePropertiesRows = [];
+
+    rows.forEach(row => {
+      const { id, osp_security_group, osp_flavor } = row;
+      minimalInstancePropertiesRows.push({
+        vm_id: id,
+        osp_security_group_id: osp_security_group.id,
+        osp_flavor_id: osp_flavor.id
+      });
+    });
+
+    input.onChange({ rows: minimalInstancePropertiesRows, updatedInstancePropertiesRowOnStandby: {} });
+  }
+
```

Comment:
Minor nit, but can we please move this method down past the `state` declaration?  Typically component lifecycle methods appear "in order", and while `state` is not exactly a lifecycle method, it used to be declared in `constructor`

Reply:
Just wanted to say that I really like this approach, of waiting to set the `redux-form` data until the user is ready to leave the step.  It feels much cleaner and more robust than updating the store per user change/interaction.

We may want to revisit this in the future and see if it makes sense to refactor the other wizard steps to follow the same pattern